generator/lookml.py

```diff
@@ -1,211 +1,50 @@
 """Generate lookml from namespaces."""
 import logging
-import re
-from itertools import filterfalse
 from pathlib import Path
-from typing import Any, Dict, Iterable, List, Tuple, TypedDict
+from typing import Dict, Iterable
```

Comment:
Moved most of the stuff in this class to either the View definition (for things that are specific to the view type, e.g. Measures) or `lookml_utils.py`, for the dimension generation (since that is the same for any view, in theory).